sql/mob_pools.sql

```diff
@@ -4796,24 +4796,24 @@ INSERT INTO `mob_pools` VALUES (4735,'Smierc','Smierc',227,0x0000340800000000000
+INSERT INTO `mob_pools` VALUES (4739,'Lancing_Wasp','Lancing_Wasp',48,0x0000900700000000000000000000000000000000,1,1,2,240,100,0,1,0,1,0,0,0,1,643,8,0,0,1,0,48);
 INSERT INTO `mob_pools` VALUES (4740,'Thrashing_Mantis','Thrashing_Mantis',340,0x000012084672656E7A6965644D616E6973000000,1,1,12,240,100,0,1,0,0,0,0,0,1,643,8,0,0,1,0,340);
 INSERT INTO `mob_pools` VALUES (4741,'Temblor_Beetle','Temblor_Beetle',49,0x00009A0100000000000000000000000000000000,7,7,8,240,100,0,1,0,1,0,0,0,1,131,0,0,0,1,0,49);
 INSERT INTO `mob_pools` VALUES (4742,'Shrubshredder_Chapuli','Shrubshredder_Chapuli',339,0x0000F10900000000000000000000000000000000,1,1,7,240,100,0,1,1,1,0,0,0,1,131,0,0,0,1,0,339);
-INSERT INTO `mob_pools` VALUES (4743,'Red_Dropwing','Red_Dropwing',113,0x0000C10142726F6164536361726C657400000000,1,1,8,240,100,0,1,0,1,0,0,0,1,643,0,0,0,1,0,113);
-INSERT INTO `mob_pools` VALUES (4744,'Quivering_twitherym','Quivering_twitherym',338,0x0000E70943617265656E69675477697468000000,1,1,12,240,100,0,1,0,1,0,0,0,7,131,0,0,0,1,0,338);
-INSERT INTO `mob_pools` VALUES (4745,'Cerise_Wasp','Cerise_Wasp',48,0x00001101556E64657267726F7774686F72700000,1,1,5,240,100,0,1,0,2,0,0,0,1,643,0,0,0,1,0,48);
+INSERT INTO `mob_pools` VALUES (4743,'Red_Dropwing','Red_Dropwing',113,0x0000C10100000000000000000000000000000000,1,1,8,240,100,0,1,0,1,0,0,0,1,643,8,0,0,1,0,113);
+INSERT INTO `mob_pools` VALUES (4744,'Quivering_Twitherym','Quivering_Twitherym',338,0x0000E70900000000000000000000000000000000,1,1,12,240,100,0,1,0,1,0,0,64,7,131,0,0,0,1,0,338);
+INSERT INTO `mob_pools` VALUES (4745,'Cerise_Wasp','Cerise_Wasp',48,0x0000110100000000000000000000000000000000,1,1,5,240,100,0,1,0,2,0,0,0,1,643,8,0,0,1,0,48);
 INSERT INTO `mob_pools` VALUES (4746,'Startled_Uragnite','Startled_Uragnite',251,0x000051054269676874557261676E696500000000,1,1,7,290,100,0,1,0,0,2,0,0,7,643,0,0,0,1,0,251);
 INSERT INTO `mob_pools` VALUES (4747,'Frondescent_Leafkin','Frondescent_Leafkin',455,0x0000E20900000000000000000000000000000000,1,1,8,240,100,0,1,0,0,0,0,0,1,129,0,0,0,1,0,455);
 INSERT INTO `mob_pools` VALUES (4748,'Shy_Heartwing','Shy_Heartwing',343,0x00000F0A00000000000000000000000000000000,4,1,11,240,100,0,0,0,0,0,0,0,1,1153,0,0,2,1,0,343);
-INSERT INTO `mob_pools` VALUES (4749,'Lavender_Twitherym','Lavender_Twitherym',338,0x0000E70943617265656E69675477697468000000,1,1,12,240,100,0,1,0,1,0,0,0,7,131,0,0,0,1,0,338);
-INSERT INTO `mob_pools` VALUES (4750,'Floodplain_Spider','Floodplain_Spider',235,0x00003801436F6C7373616C537069657200000000,1,1,7,240,100,0,1,0,1,0,0,0,7,643,0,0,0,1,0,236);
+INSERT INTO `mob_pools` VALUES (4749,'Lavender_Twitherym','Lavender_Twitherym',338,0x0000E70900000000000000000000000000000000,1,1,12,240,100,0,1,0,1,0,0,64,7,131,0,0,0,1,0,338);
+INSERT INTO `mob_pools` VALUES (4750,'Floodplain_Spider','Floodplain_Spider',235,0x0000380100000000000000000000000000000000,1,1,7,240,100,0,1,0,1,0,0,0,7,643,8,0,0,1,0,236);
```

Comment:
I'm intrigued how the previous model fields for these twitherym had random junk model data, when as "simple mobs" they should have all been "simple model" types.

Reply:
SE doesn't zero the unused bytes, so junk data gets caught in caps sometimes depending on capture methods (if you wireshark it you will get all sorts of nonsense)

Reply:
Huh, TIL.